README.md

```diff
@@ -1,19 +1,34 @@
 # docker-django
 
-[![Docker Repository on
-Quay.io](https://quay.io/repository/azavea/django/status "Docker
-Repository on Quay.io")](https://quay.io/repository/azavea/django)
-[![Apache V2
-License](http://img.shields.io/badge/license-Apache%20V2-blue.svg)](https://github.com/azavea/docker-django/blob/develop/LICENSE)
+[![Docker Repository on Quay.io](https://quay.io/repository/azavea/django/status "Docker Repository on Quay.io")](https://quay.io/repository/azavea/django)
+[![Apache V2 License](http://img.shields.io/badge/license-Apache%20V2-blue.svg)](https://github.com/azavea/docker-django/blob/develop/LICENSE)
 
-Base `Dockerfile` based off of [`debian:jessie`](https://github.com/tianon/docker-brew-debian) that installs dependencies for a Django project with geospatial support.
+A `Dockerfile` based off of [`python:2.7-slim`](https://registry.hub.docker.com/_/python/) that installs dependencies for a Django project with geospatial support.
+
+Includes:
 
-Installs the following dependencies:
   - gdal
   - postgres/psycopg2
   - gevent
   - gunicorn
 
-This images is intended to be inherited from in another Dockerfile.
+## Usage
+
+First, build the container:
+
+```bash
+$ docker build -t quay.io/azavea/django:latest .
+```
+
+Then, run the container:
 
-The image is hosted on [Quay](quay.io)
+```bash
+$ docker run --rm --entrypoint pip quay.io/azavea/django:latest freeze        
```

Comment:
Wasn't sure what to use as the example command because `gunicorn` is actually set as the `ENTRYPOINT` for this container image. Don't think anything can be done with Django without a stub project.

Since it is the Django container, I thought it might be useful to make the example command something that allows you to confirm that the correct Django version is installed.